src/gatsby-node.js

```diff
@@ -47,25 +47,37 @@ exports.sourceNodes = (
     return nodeData
   }
 
-  const { groupUrlName, ...apiOptions } = configOptions
+  const { groupUrlName, eventsOptions: paramEventsOptions, ...apiOptions } = configOptions
+  const eventsOptions = paramEventsOptions?paramEventsOptions:[apiOptions];
+
   // Convert the options object into a query string
   const queryStringOptions = queryString.stringify(apiOptions)
 
-  const apiGroupUrl = `https://api.meetup.com/${groupUrlName}?${queryStringOptions}`
-  const apiEventsUrl = `https://api.meetup.com/${groupUrlName}/events?${queryStringOptions}`
+  const apiGroupUrl = `https://api.meetup.com/${groupUrlName}?${queryStringOptions}`; // Convert the options object into a query string
```

Comment:
I think you copy-pasted the comment once too many, could you remove this one?

Reply:
@phacks , fixed copy-pasted comment that was duplicated